velox/type/StringView.h

```diff
@@ -18,11 +18,17 @@
 
 #include <functional>
 #include <string>
+#include <string_view>
 
 #include <folly/FBString.h>
+#include <folly/Format.h>
 #include <folly/Range.h>
 #include <folly/dynamic.h>
 
+#if __has_include(<fmt/format.h>)
```

Comment:
in which cases won't we have this file available to include? Wouldn't it fail compilation anyhow?

Reply:
I think when I wrote this I wasn't sure if fmt was a hard requirement for usage of StringView.h. I'll remove the guard